client/src/components/Indices/SharingIndicators.vue

```diff
@@ -44,5 +44,15 @@ const props = defineProps<SharingIndicatorsProps>();
             @click.prevent="$emit('filter', 'shared_with_me')">
             <FontAwesomeIcon icon="share-alt" />
         </BButton>
+        <!-- this can later be changed to an active button, currently, no such filter -->
+        <BButton
+            v-if="props.object.users_shared_with && props.object.users_shared_with.length"
+            v-b-tooltip.hover.noninteractive
+            size="sm"
+            variant="link"
+            disabled
+            :title="`Shared with ${props.object.users_shared_with.length} user(s)` | localize">
+            <FontAwesomeIcon icon="users" />
+        </BButton>
```

Comment:
Please resolve these typescript errors

Reply:
Can you specify the errors? This is currently not yet used, but will be in follow-ups.

Reply:
* users_shared_with unknown key
* localize unknown function
* invalid operator `|` on string (the pipe operator is custom vue syntax, but really not necessary here, and it doesn't play nice with typescript)

Are you not able to see these errors locally? Maybe there's something wrong with your ide configuration.

Reply:
Oh strange, no I don't see any errors, neither in my IDE nor when running `yarn develop`. I removed that section for now, since it is not used yet. Something to follow-up on.

Reply:
![image](https://github.com/galaxyproject/galaxy/assets/44241786/7b37a79b-f46e-4e4b-b3f5-a70e72ed03eb)